code/modules/events/wisdomcow.dm

```diff
@@ -0,0 +1,16 @@
+/datum/round_event_control/wisdomcow
+	name = "Wisdom cow"
+	typepath = /datum/round_event/wisdomcow
+	max_occurrences = 1
+	weight = 20
+
+/datum/round_event/wisdomcow/announce(fake)
+	priority_announce("A wise cow has been spotted in the area. Be sure to ask his advice.", "Nanotrasen Cow Ranching Agency")
```

Comment:
are you sure it's a "he"?

wouldn't a male cow be a steer, anyway?

Reply:
wisdom bull doesn't ring off the tongue

Reply:
yeah it would be her

Reply:
its 2020 let them identify as they want to

jk fixed it